### DISCUSSION TITLE
"Search Field" position

### DISCUSSION THREAD

**Discussion by PCO62** [Hofknecht/SystemTrayMenu#489] [Ideas]
It was more convenient to use the "**_Search Field_**" being at the **bottom** and not at the top! ;)

**Comment by Hofknecht** [comment]
@PCO62 
Thanks for your feedback!
Yes, we had it at the bottom in a past version then and when we did a major redesign of the GUI we decided to put it at the top because it is also at the top in windows 11 and is also at the top under mac finder. After much deliberation, we came to the conclusion that it is better, e.g. then the search is closer to the most likely result because it is displayed above. You could make an option to show it below but we'd better do that in v2 or later, we definitely keep the idea but not too much time right now

**Reply by PCO62** [reply]
@Hofknecht

Hi @Hofknecht

Thank you for your answer 

I understood the reasons given:
- "... closer to the most likely result because it is displayed above.
- and for being more consistent with the "layout" of the search field in Windows Explorer

Maybe it could be considered as "an option" of the app (in future updates) -choosing to be on top or be on bottom... ;)

**Comment by Hofknecht** [comment]
added label 'future extension'
search choosing to be on top or be on bottom

**Comment by PCO62** [comment]
Great  :)

Markus Hofknecht ***@***.***> escreveu no dia domingo,
27/08/2023 à(s) 15:46:

**Comment by PCO62** [comment]
Hi Markus

Regarding the SystemTrayMenu app, I had to express my opinion on the
Windows Store because the way the app was evaluated was very unfair.
Furthermore, I think that the only user who had evaluated it simply did not
know how to use it (ignorance).

So, I made my assessment (in Windows Store) which I consider fair for this
app (I use it whenever I'm using my Laptop)

Quote:
========================================
Simple and functional App

It's a simple and superb app (that overcomes the apps organization problem
in Windows 11) where we can organize and categorize all our applications or
selected folders (through shortcuts/links, just like for applications) in a
tree folder system (created by us) and which includes a search field
capable of finding any links that are in that set of folders. NOTE: to get
links to ALL installed applications (including UWP applications) simply
access the "Run" dialog (Windows key + R) and run the command
"shell:AppsFolder" (without quotes) and drag-and-drop the icons to get the
respective app links.

Paulo
08/26/2023
========================================

*Thank you, Markus, for SystemTrayMenu  **; )*

*Regards*
*Paulo Castro Oliveira*

Markus Hofknecht ***@***.***> escreveu no dia domingo,
27/08/2023 à(s) 15:46:
